src/main/java/com/internet/shop/WebShopApp.java

```diff
@@ -35,6 +40,70 @@ public static void main(String[] args) {
         testProduct.setId(cup.getId());
         productService.update(testProduct);
         System.out.println("\nAfter update by id = 0");
+
+        System.out.println("=================================================");
```

Comment:
Let's introduce a variable for such a separator

Reply:
great point

Reply:
Thx, fixed to LINE_DIVIDER